fe/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java

```diff
@@ -78,13 +74,13 @@
  * The desireTaskConcurrentNum means that user expect the number of concurrent stream load
  * The routine load job support different streaming medium such as KAFKA
  */
-public abstract class RoutineLoadJob extends TxnStateChangeListener implements Writable {
+public abstract class RoutineLoadJob implements TxnStateChangeListener, Writable {
 
     private static final Logger LOG = LogManager.getLogger(RoutineLoadJob.class);
 
     private static final int DEFAULT_TASK_TIMEOUT_SECONDS = 10;
-    private static final int BASE_OF_ERROR_RATE = 10000;
-    private static final int DEFAULT_MAX_ERROR_NUM = (int) (BASE_OF_ERROR_RATE * 0.5);
+    private static final int ERROR_SAMPLE_NUM = 1000 * 10000;
+    private static final int DEFAULT_MAX_ERROR_NUM = 0;
```

Comment:
Please do not set `DEFAULT_MAX_ERROR_NUM` to 0.  The job which is not set this value will be paused when the first commit...